WalletWasabi.Packager/Program.cs

```diff
@@ -352,7 +352,7 @@ private static void Publish()
 				{
 					string? error = process?.StandardOutput.ReadToEnd();
 					process?.WaitForExit();
-					if (process?.ExitCode is int exitCode)
+					if (process?.ExitCode is int exitCode && exitCode != 0)
```

Comment:
`process` cannot be null here. All those inventions (`process?.whatever`) are at least unnecessary or plain wrong.

Reply:
**Returns**
Process

> A new Process that is associated with the process resource, or null if no process resource is started. Note that a new process that's started alongside already running instances of the same process will be independent from the others. In addition, Start may return a non-null Process with its HasExited property already set to true. In this case, the started process may have activated an existing instance of itself and then exited.

https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.process.start?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(System.Diagnostics.Process.Start);k(DevLang-csharp)%26rd%3Dtrue&view=net-5.0

Reply:
Ohh thanks. In that case, if we get a `null` there is nothing we can do and then we should stop doing things. I mean, something like: `if (process is null) display_error_and_exit()`

Reply:
Fixed by creating a helper method.